README.md

```diff
@@ -73,8 +73,9 @@ It gets its information from a international API service that provides real-time
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-Coming soon...
-<!-- - [Live Demo Link](https://google.com) -->
+https://forex-viewer.netlify.app/
+- [Live Demo Link](https://forex-viewer.netlify.app/)
+
```

Comment:
- [ ] please ensure your Acknowledgment section has appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
done